kong/plugins/response-ratelimiting/policies/init.lua

```diff
@@ -55,6 +55,10 @@ local function get_redis_connection(conf)
   local red = redis:new()
   red:set_timeout(conf.redis_timeout)
 
+  sock_opts.ssl = conf.redis_ssl
+  sock_opts.ssl_verify = conf.redis_ssl_verify
+  sock_opts.server_name = conf.redis_server_name
+
   -- use a special pool name only if redis_database is set to non-zero
   -- otherwise use the default pool name host:port
   sock_opts.pool = conf.redis_database and
```

Comment:
```lua
  if conf.redis_database ~= 0 then
    sock_opts.pool = fmt( "%s:%d;%d",
                          conf.redis_host,
                          conf.redis_port,
                          conf.redis_database)
end
```
https://github.com/Kong/kong/blob/93ba646dc970c64a12a95950989be18f99a24ceb/kong/plugins/rate-limiting/policies/init.lua#L62-L69

I think that's better.

Reply:
done